swap/swap_test.go

```diff
@@ -808,9 +808,14 @@ func TestPeerVerifyChequeAgainstLast(t *testing.T) {
 	newCheque.Serial = oldCheque.Serial + 1
 	newCheque.Amount = oldCheque.Amount + increase
 
-	if err := peer.verifyChequeAgainstLast(newCheque, oldCheque, increase); err != nil {
+	actualAmount, err := peer.verifyChequeAgainstLast(newCheque, oldCheque, increase)
+	if err != nil {
 		t.Fatalf("failed to verify cheque compared to old cheque: %s", err.Error())
 	}
+
+	if actualAmount != increase {
+		t.Fatalf("incr")
```

Comment:
Can we have a better error message here?
Also, only use `Fatalf` if you are going to format, otherwise just `t.Fatal`